assets/src/edit-story/components/dropDown/index.js

```diff
@@ -0,0 +1,300 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useState, useEffect, useCallback, useMemo, useRef } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { rgba } from 'polished';
+import { debounce } from 'throttle-debounce';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ReactComponent as DropDownIcon } from '../../icons/dropdown.svg';
+import { useKeyDownEffect } from '../keyboard';
+import useFocusOut from '../../utils/useFocusOut';
+
+const DropDownContainer = styled.div`
+  width: 100px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  color: ${({ theme }) => theme.colors.fg.v0};
+  font-family: ${({ theme }) => theme.fonts.body1.font};
+`;
+
+const DropDownSelect = styled.div.attrs({ role: 'Listbox', tabIndex: '0' })`
```

Comment:
Nit: Should the role be lowercase?

Reply:
all role updated with button, listbox, option..